runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -62,31 +42,195 @@
+	CITY_SOPHANEM("Sophanem" , DiscordAreaType.CITIES, ImmutableList.of(13099)),
+	CITY_TAI_BWO_WANNAI("Tai Bwo Wannai" , DiscordAreaType.CITIES, ImmutableList.of(11056, 11055)),
+	CITY_TAVERLEY("Taverley" , DiscordAreaType.CITIES, ImmutableList.of(11574, 11573)),
+	CITY_TREE_GNOME_STRONGHOLD("Tree Gnome Stronghold" , DiscordAreaType.CITIES, ImmutableList.of(9782, 9781)),
+	CITY_TREE_GNOME_VILLAGE("Tree Gnome Village" , DiscordAreaType.CITIES, ImmutableList.of(10033)),
+	CITY_TROLL_STRONGHOLD("Troll Stronghold" , DiscordAreaType.CITIES, ImmutableList.of(11321)),
+	CITY_TYRAS_CAMP("Tyras Camp" , DiscordAreaType.CITIES, ImmutableList.of(8753, 8752)),
+	CITY_UZER("Uzer" , DiscordAreaType.CITIES, ImmutableList.of(13872)),
+	CITY_VARROCK("Varrock" , DiscordAreaType.CITIES, ImmutableList.of(12596, 12597, 12598, 12852, 12853, 12854, 13108, 13109, 13110)),
+	CITY_WITCHHAVEN("Witchaven" , DiscordAreaType.CITIES, ImmutableList.of(10803)),
+	CITY_YANILLE("Yanille" , DiscordAreaType.CITIES, ImmutableList.of(10288, 10032)),
+	CITY_ZANARIS("Zanaris" , DiscordAreaType.CITIES, ImmutableList.of(9285, 9541, 9540, 9797)),
+	CITY_ZULANDRA("Zul-Andra" , DiscordAreaType.CITIES, ImmutableList.of(8751)),
+
+	// Dungeons
+	DUNGEON_ABANDONED_MINE("Abandoned Mine", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_AH_ZA_RHOON("Ah Za Rhoon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_ANCIENT_CAVERN("Ancient Cavern", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_APE_ATOLL("Ape Atoll Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_ARDY_SEWERS("Ardougne Sewers", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_ASGARNIAN_ICE_CAVES("Asgarnian Ice Caves", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_BRIMHAVEN("Brimhaven Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_BRINE_RAT_CAVERN("Brine Rat Cavern", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_CATACOMBS_OF_KOUREND("Catacombs of Kourend", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_CHASM_OF_FIRE("Chasm of Fire", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_CLOCK_TOWER("Clock Tower Basement", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_CORSAIR_COVE("Corsair Cove Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_CRABCLAW_CAVES("Crabclaw Caves", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_DIGSITE("Digsite Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_DORGESHKAAN("Dorgesh-Kaan South Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_DORGESHUUN_MINES("Dorgeshuun Mines", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_DRAYNOR_SEWERS("Draynor Sewers", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_DWARVEN_MINES("Dwarven Mines", DiscordAreaType.DUNGEONS, ImmutableList.of()),
+	DUNGEON_EAGLES_PEAK("Eagles' Peak Dungeon", DiscordAreaType.DUNGEONS, ImmutableList.of()),
```

Comment:
Add new constructor override without last param, all these empty immutable lists are weird

Reply:
These should all be filled with RegionIds though. Any which will not contain a RegionId I will remove.

Reply:
Ahh